File: src/test/java/net/slipp/social/security/SlippUserDetailsServiceTest.java
package net.slipp.social.security;

import static org.hamcrest.CoreMatchers.*;
import static org.junit.Assert.*;

import org.junit.Test;

public class SlippUserDetailsServiceTest {

	@Test
	public void isAdmin() {
		String adminUsers = "javajigi:sanjigi";
		SlippUserDetailsService dut = new SlippUserDetailsService();
		dut.setAdminUsers(adminUsers);
		assertThat(dut.isAdmin("javajigi"), is(true));
		
		assertThat(dut.isAdmin("userId"), is(false));
	}

}

File: src/test/scala/net/slipp/social/security/SlippUserDetailsServiceTest.scala
package net.slipp.social.security

import org.hamcrest.CoreMatchers._
import org.junit.Assert._
import org.junit.Test

class SlippUserDetailsServiceTest {
  @Test def isAdmin {
    val adminUsers: String = "javajigi:sanjigi"
    val dut: SlippUserDetailsService = new SlippUserDetailsService
    dut.setAdminUsers(adminUsers)
    assertThat(dut.isRootAdmin("javajigi"), is(true))
    assertThat(dut.isRootAdmin("userId"), is(false))
  }
}
